src/__stories__/Index.stories.mdx

```diff
@@ -1,38 +1,6 @@
-import { Meta } from '@storybook/addon-docs'
+import { Meta, Description } from '@storybook/addon-docs'
+import Readme from '../../README.md'
 
-<Meta title="Home" />
+<Meta title="Get started" />
 
-# Welcome to Scaleway UI
-
-Scaleway UI is a set of React components you can use to build your application. It is used to build our [Scaleway Console](https://console.scaleway.com).
-
-## Getting started
-
-### Installation
-
-```bash
-pnpm add @scaleway/ui
-```
-
-### Add a ThemeProvider
-
-To use the library you need to put a `ThemeProvider` which come from `@emotion/react` library:
-
-```jsx
-import { ThemeProvider } from '@emotion/react'
-import { Text, theme } from '@scaleway/ui'
-
-export default function App() {
-  return (
-    <ThemeProvider theme={theme}>
-      <Text as="h1" variant="headingLarge">
-        Welcome to Scaleway UI
-      </Text>
-    </ThemeProvider>
-  )
-}
-```
-
-### Customizing
-
-The default theme is available on Github, you can see it [here](https://github.com/scaleway/scaleway-ui/blob/main/src/theme/index.ts). For more customization go to [Colors page](/docs/theme-colors--palette)
+<Description markdown={Readme} />
```

Comment:
No way to interpret emoji ? 

Also why remove the quickstart ? Changelog should be on another page no ?